src/plugins/intel_gna/gna_device.cpp

```diff
@@ -111,6 +109,23 @@ void GNADeviceHelper::setUpActiveList(const uint32_t requestConfigId, uint32_t l
     checkGna2Status(status, "Gna2RequestConfigEnableActiveList");
 }
 
+void GNADeviceHelper::enableDiagnostics() {
```

Comment:
what if we remove debugLogEnabled member and just create a method for checking value.
```
bool isDiagnosticEnabled()
{
    return (gnaFlags->log_level == ov::log::Level::DEBUG);
}
```

Reply:
Currently `gnaFlags` are outside `GNADeviceHelper`, we can consider it in future with more refactoring.